packages/composer/amazeelabs/silverback_gutenberg/tests/src/Kernel/LinkProcessorTest.php

```diff
@@ -122,6 +122,8 @@ public function testProcessUrl() {
               'en' => '/node/' . $withoutAlias->id(),
               'de' => '/de/node/' . $withoutAlias->id(),
             ],
+          ],
+          [
```

Comment:
The automated tests had some wrongly closed array for the outbound cases

Reply:
I pushed a change that simplifies this monster :)